amalgamation/Makefile

```diff
@@ -16,6 +16,9 @@ else
 	DEFS+=-DMSHADOW_USE_SSE=0
 endif
 
+ifndef APPLE
+    export APPLE=0
```

Comment:
Thanks for reminding, I'll remove this then, changes would be submitted with conflict resolved.